examples/cpp/main.cpp

```diff
@@ -36,6 +36,9 @@ int main() {
   printf("Reading ADC Channels: 1 = %i, 2 = %i, 3 = %i, 4 = %i\n",
          adc.channel[0], adc.channel[1], adc.channel[2], adc.channel[3]);
 
+  printf("Data ADC Channels: 1 = %i\n", read_adc(adc_Channel::Ch1)
+    );
+
```

Comment:
that's a weird formatting

Reply:
Check https://github.com/bluerobotics/navigator-lib/pull/16 to avoid such problems